### ISSUE TITLE
Update/cron.php does not exist

### ISSUE THREAD

**Issue by camdixon** [magento/devdocs#1740]
Feedback on page: /guides/v2.2/config-guide/cli/config-cli-subcommands-cron.html

When I installed my cron from this page, it generated this crontab as my user which the docs said it would.  However, the update/cron.php does not exist.  Since it appends to a log file at the bottom I give you the output of my log file.

```
#~ MAGENTO START
* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
#~ MAGENTO END
```

```
cat <magento2_root>var/log/update.cron.log
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
Could not open input file: <magento2_root>/update/cron.php
```

**Comment by hostep** [comment]
@camdixon: this thread might contain some useful info for you: https://github.com/magento/magento2/issues/10040#issuecomment-311775893

**Comment by camdixon** [comment]
Yes, it is useful, but then what about the dev docs for the cron that does the update?  @magento-engcom-team  didn't make a decision about this yet it seems.  The command line cron installer adds this update.  Also, I'm now curious is if it's actually required?

**Comment by meker12** [comment]
We are re-organizing our unresolved GH issues into smaller, more focused projects, and have moved this to the  Installation and Configuration Documentation project.

**Comment by atwixfirster** [comment]
Explanations provided in https://github.com/magento/devdocs/pull/5298

Thank you for help , @hostep

**Comment by jeff-matthews** [comment]
Thanks @atwixfirster! We'll review next week.

**Comment by atwixfirster** [comment]
> Thanks @atwixfirster! We'll review next week.

sounds pretty optimistic, @jeff-matthews 😆 

Many thanks

**Comment by dobooth** [comment]
This is fixed in 2.3.4. We need to investigate this to see what still needs updating in this topic. @atwixfirster, would you be able to look into this?

**Comment by atwixfirster** [comment]
> This is fixed in 2.3.4. We need to investigate this to see what still needs updating in this topic. @atwixfirster, would you be able to look into this?

Sure, @dobooth

**Comment by atwixfirster** [comment]
@dobooth , 

Per my opinion you can close this issue because of merged:
- https://github.com/magento/magento2/pull/24187
- https://github.com/magento/devdocs/pull/6428

From my side, there is nothing to add here.

Thank you!
